core/src/main/java/io/github/tcdl/msb/impl/ObjectFactoryImpl.java

```diff
@@ -42,4 +50,18 @@ public Requester createRequester(String namespace, RequestOptions requestOptions
     public ResponderServer createResponderServer(String namespace,  MessageTemplate messageTemplate, ResponderServer.RequestHandler requestHandler) {
         return ResponderServerImpl.create(namespace, messageTemplate, msbContext, requestHandler);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ChannelMonitorAggregator createChannelMonitorAggregator(Callback<AggregatorStats> aggregatorStatsHandler) {
+        ThreadFactory threadFactory = new BasicThreadFactory.Builder()
+                .namingPattern("monitor-aggregator-heartbeat-thread-%d")
```

Comment:
Defensive technique. It's better to have extra "1" in the thread name than have identical thread names if we increase the pool size in future for some reason.